build/ci/evergreen.yml

```diff
@@ -1527,5 +1510,29 @@ buildvariants:
       <<: *go_linux_version
     tasks:
       - name: ".e2e .deployments .atlas"
+  - name: e2e_local_deployments
```

Comment:
can we have one `e2e_local_deployments` that runs on waterfall for all commits and then `e2e_local_deployments_multi_arch` to run everyday maybe?